opendrift_leeway_webgui/simulation.py

```diff
@@ -107,11 +115,177 @@ def main():
     simulation.run(
         duration=timedelta(hours=args.duration), time_step=600, outfile=f"{outfile}.nc"
     )
-    simulation.plot(
-        fast=True, legend=True, filename=f"{outfile}.png", linecolor="age_seconds"
-    )
+
+    #Plotting results
+    lon,lat =np.array(simulation.get_lonlats())
+    lon[lon==0]=np.nan
+    lat[lat==0]=np.nan
+
+    crs=ccrs.Mercator() #Mercator projection to have angle true projection
+    gcrs = ccrs.PlateCarree(globe=crs.globe) #PlateCarree for straight lines
+
+    fig=plt.figure(figsize=(8,8)) #figsize set low to get small files
+    ax = plt.axes(projection=crs) 
+
+    # base map layer
+    ax.add_wms(wms='https://sgx.geodatenzentrum.de/wms_topplus_open',layers=['web'])#quote source: Kartendarstellung: © Bundesamt für Kartographie und Geodäsie (2021), Datenquellen: https://gdz.bkg.bund.de/index.php/default/wms-topplusopen-wms-topplus-open.html
```

Comment:
I think the source of the map needs to be added to the image/mail. It is probably not sufficient to add it in the source code?

Reply:
https://github.com/digitalfabrik/opendrift-leeway-webgui/pull/95/files#diff-b663ceaec59827ae2ee9363fda9f88c500c2d995eaf0dfea31f62213aaa40e91R260

Reply:
It is in the bottom left of the image